assignments/prototypes.js

```diff
@@ -15,13 +15,28 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject (createdAt,name, dimensions) {
```

Comment:
Both these constructor (line 18 and line 24) are not working properly because you have the wrong parameters set up. 

If you notice the mage is instantiated passing a single object as an argument (with all the attributes inside), but these constructors are built to receive more arguments then just one. 
You could want to fix that. 
(Btw in your Humanoid constructor you are doing it in the right way)

If you have problems figuring out how to do that let me know.